.github/workflows/sanity-checks.yml

```diff
@@ -40,10 +37,10 @@ jobs:
 
   benchmark-check:
     name: bench-check-${{ matrix.runtime }}
-    runs-on: ubuntu-latest #r-4-cores
+    runs-on: ubuntu-latest #-4-cores
     strategy:
       matrix:
-        runtime: [development, altair, centrifuge]
+        runtime: [altair, centrifuge]
```

Comment:
Why not development?

Reply:
I thought we didnt need to check dev benchmarks to save time. I know this kind of contradicts my argument of keeping benchmarks in Dev. However, in the past releases I copy pasted Centrifuge benchmarks to Dev because there was no runtime benchmark pipeline. I think thats feasible because this way we mimic mainnet.

Reply:
Since this are PR checks, let's keep it minimal, this won't actually publish or set benchmarks anywhere, it's just to check that the benchmarks run
Do we really need all 3 to run?